CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 ### Feature
 
+- Refactor CreateContent command to accept a single json object @iFlameing
```

Comment:
@iFlameing nitpick: createContent is lower camel case and we should not that this is a Cypress function.

Reply:
done